### ISSUE TITLE
FC 20231012 is incompatible with S-100 5.x

### ISSUE THREAD

**Issue by rmalyankar** [iho-ohi/S-129-Product-Specification#1]
The XML FC currently in the [20231012](https://github.com/iho-ohi/S-129-Product-Specification/tree/main/1.1.0/FC/20231012) folder does  not validate  with the S-100 5.0.0 FC schema, the namespaces and schemaLocation have not been updated. The files in the "pre-1.1.0" folder appear to have them right (though I checked only the 20230727 file).

The version number of the FC should correspond to the S-129 edition, not the S-100 edition.

**Comment by jrhee01** [comment]
thanks @rmalyankar. The namespaces and schemaLocation should be consistent across the 3 under the "pre-1.1.0" folder.
I will look at uploading a new copy consistent with the 20231012 version, but with the namespaces and schemaLocation corrected.

**Comment by jrhee01** [comment]
FC initially under the 20231012 folder has been renamed and relocated to "1.1.0/FC/S-129_1_1_0_FC.XML".
Updates have then been made to this FC for the namespaces and schemaLocation.

**Comment by rmalyankar** [comment]
There are other schema-validation problems, I corrected them as I understood the intent and submitted a pull request with the corrected file. Up to you @jrhee01 to review the differences and accept or reject as you think best.

Please note, I have not checked anything other than whether it validates with the S-100 5.0.0 FC schema.

**Comment by jrhee01** [comment]
many thanks for the pick ups @rmalyankar. I should've seen your comment first before pushing my own changes, as I also noticed:

1. mismatch b/w "UnderKeelClearancePurpose" vs "UnderKeelClearancePurposeType", 
2. mismatch b/w "UnderKeelClearanceCalculationRequested" vs "UnderKeelClearanceCalculationRequested". 
3. letter inconsistency in the 2 attribute names

The actual attribute names are without "Type", so I have merged our changes accordingly.

Thanks for adding S100FC:productId.
